### PR TITLE
Create main entry point index.js with a single point of configuration

### PR DESCRIPTION
**Pull request by guillett**

### CODE REVIEW

**Comment by guillett** [comment]
I successfully used this version in mes-aides-ui with `babel-node`.

**Comment by MattiSG** [comment]
Fetching the configuration data from the `package.json` was a design choice. That was expected to make it easier to configure Ludwig in any client. Let's discuss IRL why this decision had to be reverted.

**Comment by guillett** [comment]
OK, to add some context:
- https://github.com/sgmap/ludwig/pull/60#r64905918
- #67

**Comment by guillett** [comment]
Plus, IMHO, the usage of a GitHub application with clientID/clientSecret is overkill in a POC.
A user [`accessToken`](https://github.com/blog/1509-personal-api-tokens) (cf. [your](https://github.com/settings/tokens)) is enough and allow anyone to contribute.

**Comment by guillett** [comment]
Utiliser `npm config` me parait plus contraignant que le faire via un fichier de configuration.

Pour versionner publiquement le fichier de configuration sans les secrets, la stratégie suivante peut être adoptée :
```js
module.exports = {
    [...]
    github: {
        accessToken: process.env.LUDWIG_GITHUB_ACCESS_TOKEN,
        [...]
    },
    [...]
};
```

File: index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    app.set('view engine', 'ejs');
+    app.use(express.static(path.join(__dirname, '/dist')));
+    app.use(bodyParser.urlencoded({ extended: false }));
+
+    app.use(session({
+        secret: appConfiguration.session.secret,
+        resave:false,
+        saveUninitialized:false,
+        store:new MongoStore({mongooseConnection: mongoose.connection})
```
Comment (MattiSG) [inline index.js]: > Inconsistent whitespace: spaces after colons.
>
> — ok, this code was just moved, not blocking.

**Comment by MattiSG** [comment]
As discussed IRL yesterday: ok for using a config object. If a user considers NPM config is the best fit for secret management, she can always load the corresponding env variable in the config.

**Pull request merged**